.github/renovate.json

```diff
@@ -81,7 +81,14 @@
       "matchStrings": ["ARG DOCKER_VERSION=(?<currentValue>.*?)\n"],
       "depNameTemplate": "docker",
       "datasourceTemplate": "docker"
+    },
+    {
+      "fileMatch": ["src/test/java/plugins/OicAuthPluginTest.java"],
+      "matchStrings": [".* KEYCLOAK_IMAGE =\n\\s*\"(?<repo>.*?):(?<currentValue>.*?)@(?<currentDigest>sha256:.*?)\";\n"],
+      "depNameTemplate": "{{{repo}}}",
+      "datasourceTemplate": "docker"
```

Comment:
`LOG_LEVEL=debug RENOVATE_CONFIG_FILE=/usr/src/app/.github/renovate.json renovate --platform=local` shows this will match:

```
             "deps": [
               {
                 "depName": "keycloak/keycloak",
                 "currentValue": "25.0.4",
                 "currentDigest": "sha256:bf788a3b7fd737143f98d4cb514cb9599c896acee01a26b2117a10bd99e23e11",
                 "datasource": "docker",
                 "replaceString": "    private static final String KEYCLOAK_IMAGE =\n            \"keycloak/keycloak:25.0.4@sha256:bf788a3b7fd737143f98d4cb514cb9599c896acee01a26b2117a10bd99e23e11\";\n",
                 "updates": [
                   {
                     "bucket": "non-major",
                     "newVersion": "25.0.5",
                     "newValue": "25.0.5",
                     "newDigest": "sha256:410fce4b9b40e1f4e7f90b44acedbaa6d935bacea432a6884769067b253d46fb",
                     "releaseTimestamp": "2024-09-10T05:21:31.638Z",
                     "newMajor": 25,
                     "newMinor": 0,
                     "newPatch": 5,
                     "updateType": "patch",
                     "branchName": "renovate/keycloak-keycloak-25.x"
                   }
                 ],
                 "packageName": "keycloak/keycloak",
                 "versioning": "docker",
                 "warnings": [],
                 "sourceUrl": "https://github.com/keycloak-rel/keycloak-rel",
                 "registryUrl": "https://index.docker.io",
                 "currentVersion": "25.0.4",
                 "currentVersionTimestamp": "2024-08-19T09:21:17.507Z",
                 "isSingleVersion": true,
                 "fixedVersion": "25.0.4"
               }
```